lib/entities/quote/DutchQuote.ts

```diff
@@ -177,11 +188,16 @@ export class DutchQuote implements IQuote {
       quote.swapper,
       quote.quoteType,
       quote.filler,
-      quote.nonce
+      quote.nonce,
+      // Prefers the portion from the RFQ quoter if exists
+      // Next the potion from the classic quoter if exists
+      // Otherwise no portion
+      quote.portionBips ?? classic.getPortionBips(),
```

Comment:
shouldnt these values always be the same?

Reply:
Note flow is 
for RFQ: DutchQuote.fromResponseBody() -> DutchQuote.reparameterize (after classic is done)
for Open quotes: DutchQuote.fromClassicQuote()

So in either case we have access to the portion bps from classic eventually, so maybe dont even need to fetch it from rfq qutoer

Reply:
mark i made the same comment to siyu offline, he mentioned its for defensive programming purposes and indeed they should never diverge

Reply:
I don't understand how this is defensive programming by duplicating API requests that should never diverge? what are we defending against?

Reply:
I am having a hard time understanding what are the duplicate API requests, can you elaborate on that? This code just reads a pre-existing field from classic, no extra calls are being made?

Its defensive in case for some reason `quote.portionBips` had a blip and it was not calculated correctly (for example the service that it depends on sent a random 500 on that request), so we can pick it up form `classic.getPortionBips()`

Reply:
oh I'm referring to my comment here for the API call duplication https://github.com/Uniswap/unified-routing-api/pull/271#discussion_r1346265017 since if we just always used the classic portionBps here we wouldn't even need to fetch it separately for the dutch part (left hand side of this coalesce)